### ISSUE TITLE
Socket.IO connection from HTTPS Admin to Web fails with CORS error

### ISSUE THREAD

**Issue by AlCalzone** [ioBroker/ioBroker.web#23]
It doesn't matter if the web instance has HTTPS or not. If the Admin runs without HTTPS, everything is fine.
![grafik](https://user-images.githubusercontent.com/17641229/62312698-283e3800-b48f-11e9-97ce-663f2b56c1db.png)

**Comment by Apollon77** [comment]
Can you please retest with newest admin and web?!

**Comment by AlCalzone** [comment]
Yep, still happens. This can only be fixed if the web adapter explicitly sets a CORS header for the admin port.

latest web:
![grafik](https://user-images.githubusercontent.com/17641229/84360844-06433500-abcb-11ea-8302-36198e109927.png)

latest admin:
![grafik](https://user-images.githubusercontent.com/17641229/84360874-14915100-abcb-11ea-8c68-43559dd19be5.png)

and I opened the settings of iQontrol 0.4.1, which tries to read something from the web instance:
![grafik](https://user-images.githubusercontent.com/17641229/84361006-4efaee00-abcb-11ea-83e5-56e7e82b8172.png)
